extensions/parquet/table/src/main/java/io/deephaven/parquet/table/MappedSchema.java

```diff
@@ -26,21 +26,14 @@ static MappedSchema create(
             final TableDefinition definition,
             final RowSet rowSet,
             final Map<String, ? extends ColumnSource<?>> columnSourceMap,
-            final ParquetInstructions instructions,
-            final ColumnDefinition<?>... extraColumns) {
```

Comment:
What do we think this was for? Maybe imposing extra columns for compatibility with a partitioned schema? Does enterprise use it?